examples/connect-electron/src/main/index.ts

```diff
@@ -1,12 +1,93 @@
 import { join } from "node:path";
 import { electronApp, is, optimizer } from "@electron-toolkit/utils";
-import { getTreasureLauncherAuthToken } from "@treasure-dev/launcher";
-import { BrowserWindow, app, ipcMain, shell } from "electron";
+import { BrowserWindow, app, ipcMain, session, shell } from "electron";
 import icon from "../../resources/icon.png?asset";
 import { startRedirectApp } from "./app";
 
 let mainWindow: BrowserWindow;
 
+export function getTreasureLauncherAuthToken(): string | undefined {
+  let args: string[] | undefined;
+
+  if (typeof process !== "undefined" && Array.isArray(process.argv)) {
+    args = process.argv;
+  } else if (
+    typeof window !== "undefined" &&
+    window.process &&
+    Array.isArray(window.process.argv)
+  ) {
+    args = window.process.argv;
+  } else {
+    return undefined;
+  }
+
+  const arg = args.find((arg) => arg.startsWith("--tdk-auth-token="));
+  return arg ? arg.split("=")[1] : undefined;
+}
+
+export function getTreasureLauncherWalletComponents():
```

Comment:
Can we reuse the util from the package instead of mostly duplicating here?

Reply:
Getting this to work was a pain but figured it out. Was having issues with the import resolving when building